Code/GeorgiaLibrarySystem-/GtlService/DataManagement/TestClass.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using GtlService.Model;
+
+namespace GtlService.DataManagement
+{
+    class TestClass
+    {
+        /* Example */
+        public static void Main(string[] args)
```

Comment:
why are you doing DataAcess stuff in the DM layer?

Reply:
About DM it was a mistake, ill fix it (good that there are reviews :D) and yes it is the db-first approach